spring-cloud/spring-cloud-openfeign/src/test/java/com/baeldung/cloud/openfeign/OpenfeignUnitTest.java

```diff
@@ -0,0 +1,50 @@
+package com.baeldung.cloud.openfeign;
+
+import com.baeldung.cloud.openfeign.client.JSONPlaceHolderClient;
+import com.baeldung.cloud.openfeign.model.Post;
+import com.baeldung.cloud.openfeign.service.JSONPlaceHolderService;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.List;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class OpenfeignUnitTest {
+
+    @Autowired
+    private JSONPlaceHolderService jsonPlaceHolderService;
+
+    @Test
+    public void whenSpringContextIsBootstrapped_thenNoExceptions() {
+    }
+
+    @Test
+    public void whenGetPosts_thenListPostSizeGreaterThanZero() {
+
+        List<Post> posts = jsonPlaceHolderService.getPosts();
+
+        assert posts.size() > 0;
```

Comment:
The assertions in this test are using `assert` as opposed to JUnit's `assertEquals` or `assertThat`

I would recommend moving to using the assertions library from JUnit. See `org.junit.Assert`.

For testing the size of a list, I'd recommend something like `assertFalse(posts.isEmpty())`